dart/constraint/DantzigLCPSolver.cpp

```diff
@@ -63,13 +63,15 @@ DantzigLCPSolver::~DantzigLCPSolver()
 //==============================================================================
 void DantzigLCPSolver::solve(ConstrainedGroup* _group)
 {
-  // If there is no constraint, then just return true.
-  size_t numConstraints = _group->getNumConstraints();
-  if (numConstraints == 0)
-    return;
 
   // Build LCP terms by aggregating them from constraints
+  size_t numConstraints = _group->getNumConstraints();
   size_t n = _group->getTotalDimension();
+
+  // If there is no constraint, then just return.
+  if (n == 0)
```

Comment:
nitpick: some compiler would complain to comparing signed and unsigned. `if (0u == n)` would help. also, @mxgrey would like to use [`Yoda conditions`](https://en.wikipedia.org/wiki/Yoda_conditions). :wink: